### ISSUE TITLE
[Question] Is there a way to check that a language code is valid?

### ISSUE THREAD

**Issue by lakemist** [LuminosoInsight/langcodes#28]
Using Language.make(x) or Language.get(x) always return a Language object.  What is the best way to determine if that object is a valid language?

e.g. Users often enter 'jp' instead of 'ja' for Japanese.  But 'jp' is not a valid code.
```python
>>> Language.get('jp')
Language.make(language='jp')

>>> Language.get('jp').language_name(min_score=100)
'jp'
```
Would the following be correct?  It seems very awkward at best to rely on this.
```python
def is_valid(lang, min_score=75):
    return lang.language_name(min_score=min_score) != lang.language
```

**Comment by jamescooke** [comment]
I'm also looking for an easy way to validate that a string is valid BCP47.

When input is `"_____"`, then it's easy because an exception is thrown using `standardize_tag()`:

```py
In [1]: from langcodes import standardize_tag
In [2]: standardize_tag("____")
...
LanguageTagError: Expected a language code, got ''
```

But this does not reliably throw exceptions, for example:

```py
In [3]: standardize_tag("ASDFASDF")
Out[3]: 'asdfasdf'
```

And the `"jp"` case is definitely *not* covered:

```py
In [4]: standardize_tag("jp")
Out[4]: 'jp'
```

Could there be a `strict=True` flag added to `standardize_tag()` to ensure that it raises for cases where no valid language can be found in the input?

**Comment by plammens** [comment]
This is an essential feature; this library isn't very useful without it.

**Comment by rspeer** [comment]
Thanks for the request -- this is now available in langcodes 3.0.
